PeerioTechnologies/peerio-mobile
# feat: action sheet slide in/out animation [branch ch7168]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/7168/mobile-use-slide-in-animation-for-action-sheet
#### Testing instructions  
Verify that showing files action sheet works properly with slide in animation.
Verify that cancelling the action by pressing cancel or by pressing outside the action sheet hides the action sheet with the correct slide out animation.
Verify that pressing on action buttons correctly dismisses action sheet and performs the correct action.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
